README.md

```diff
@@ -4,62 +4,46 @@ A SBOM-centric security scanner for Kubernetes.
 
 This is still being developed. For additional details, please refer to the [RFC](docs/rfc).
 
-# Development
+# Installation
 
-## Run tests
+To install SBOMbastic in your Kubernetes cluster, you can use the provided Helm chart.
 
-```shell
-make test
-```
-
-## Run the development environment with Tilt
-
-Customize `tilt-settings.yaml` to your needs.
-A default StorageClass must be defined in your cluster for SQLite data persistence.
-
-Run tilt:
+Install cert-manager if you don't have it already:
 
 ```shell
-tilt up
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace
```

Comment:
```suggestion
helm install cert-manager jetstack/cert-manager \
  --namespace cert-manager \
  --create-namespace \
  --set crds.enabled=true
```

This is needed, because by default the CRDs are not installed by the helm chart. See [upstream docs](https://cert-manager.io/docs/installation/helm/)